lib/utils.js

```diff
@@ -9,7 +9,12 @@ const assert = require("node:assert");
 const SUPPORTED_TEST_IDENTIFIERS = new Set(["test", "asyncTest", "only"]);
 
 const OLD_MODULE_HOOK_IDENTIFIERS = ["setup", "teardown"];
-const NEW_MODULE_HOOK_IDENTIFIERS = ["beforeEach", "afterEach"];
+const NEW_MODULE_HOOK_IDENTIFIERS = [
+    "before",
+    "beforeEach",
+    "afterEach",
+    "after",
+];
```

Comment:
Would it make sense to split this change into a separate PR? It seems there's a bug fix to `no-assert-equal` as mentioned in the PR title, but then there's a separate change to the hooks used by separate rules.